src/4.3_to_5.0/addLegacyBookmarkToUIFolder.ts

```diff
@@ -0,0 +1,61 @@
+import { ContentRecord } from "@activeviam/activeui-sdk-5.1";
+import { produce } from "immer";
+
+/**
+ * Returns a new `legacyUIFolder` with the given `legacyBookmarksToAdd` added.
+ * Useful for creating legacyUIFolders for tests.
+ * Does not mutate `legacyUIFolder`.
+ */
+export function addLegacyBookmarkToUIFolder(
```

Comment:
This could take a `contentType` argument in order to be able to add saved dashboards, but also widgets or filters into the legacy UI folder.

Reply:
This is for legacy "bookmarks" and back in the day we used to plop everything into the `bookmarks` folder, so no need to differentiate 👀